jwtmiddleware.go

```diff
@@ -160,58 +232,45 @@ func FromFirst(extractors ...TokenExtractor) TokenExtractor {
 	}
 }
 
-func (m *JWTMiddleware) CheckJWT(w http.ResponseWriter, r *http.Request) error {
-	if !m.Options.EnableAuthOnOptions {
-		if r.Method == "OPTIONS" {
+func (m *JWTMiddleware) CheckJWT(r *http.Request) error {
+	if !m.validateOnOptions {
+		if r.Method == http.MethodOptions {
 			return nil
 		}
 	}
 
-	// Use the specified token extractor to extract a token from the request
-	token, err := m.Options.Extractor(r)
+	token, err := m.tokenExtractor(r)
 
-	// If debugging is turned on, log the outcome
 	if err != nil {
-		m.logf("Error extracting JWT: %v", err)
-	} else {
-		m.logf("Token extracted: %s", token)
-	}
+		// m.logf("Error extracting JWT: %v", err)
```

Comment:
you'll notice a lot of these debug logs commented out - that's because I commented everything out and worked through testing from there. Debug pieces still need to be determined.

Reply:
you mean how to log? or what to log?